### PR TITLE
Refactoring of various things, mostly names and locations.

### PR DESCRIPTION
**Pull request by jenkoian**
* Tweaks the naming and file locations a little bit to make them more
  consistent with regular psr-4 loaded projects
* Introduced php-cs-fixer for automated coding standard fixing
* Tweaked some of the tests
* Autoloaded the utils function

### CODE REVIEW

File: src/Util/util.php [hunk 1]
```diff
@@ -22,18 +20,19 @@
...
-	// Strip the common roots from both arrays.
-	while( count( $arFrom ) && count( $arTo ) && ( $arFrom[0] == $arTo[0] ) ) {
-		array_shift( $arFrom );
-		array_shift( $arTo );
-	}
-	// for any items left in from, add '../' and then append the remaining
-	// to items.
-	return str_pad( '', count( $arFrom ) * 3, '..' . $ps ) . implode( $ps, $arTo );
+function rel_path($from, $to, $ps = DIRECTORY_SEPARATOR): string
+{
```
Comment (andrewjt71) [inline src/Util/util.php:24]: Did you put this `{` on a newline accidentally?

File: src/Util/util.php [hunk 2]
```diff
@@ -43,6 +42,7 @@ function rel_path( $from, $to, $ps = PS ): string {
  * @param  string $ds   The directory separator to standardize on
  * @return string       The normalized path
  */
-function normalize( $path, $ds ): string {
-	return str_replace( array( '/', "\\" ), $ds, $path );
+function normalize($path, $ds): string
+{
```
Comment (andrewjt71) [inline src/Util/util.php:46]: Again, any reason why `{` on a new line?

Comment (andrewjt71) [reply src/Util/util.php:46]: Oh, they all are ... looks like this is on purpose then :)

File: src/Util/util.php (hunk 1 above)
Comment (andrewjt71) [reply src/Util/util.php:24]: Oh, they all are ... looks like this is on purpose then :)

Comment (jenkoian) [reply src/Util/util.php:24]: Yeah added php-cs-fixer and set it to PSR-12 (https://github.com/boxuk/wp-muplugin-loader/pull/21/files#diff-8ec5bae185dc9af5b5bfdcd5f1dcbee62f18525ee488996d52b22729f4076ef0R12)

**Review by andrewjt71 — APPROVED** [review]
Had a skim and LGTM

**Pull request merged**